client/blocks/eligibility-warnings/warning-list.tsx

```diff
@@ -46,7 +46,7 @@ export const WarningList = ( { context, translate, warnings, showContact = true
 							/>
 							{ domainNames && displayDomainNames( domainNames ) }
 							{ supportUrl && (
-								<ExternalLink href={ supportUrl } target="_blank" rel="noopener noreferrer">
+								<ExternalLink href={ supportUrl } target="_blank">
```

Comment:
While I wasn't planning on this, this refactor did dig up some inconsistencies in pre-existing types, so I fixed them in 3ac2e34989513c69f63283fe99fa4b81d8af853f, and the most instances were like this: providing this `rel="noopener noreferrer" prop when `ExternalLink` would ignore it and provide it automatically. Thank you, types.

Reply:
Interesting, it turns out that the `rel` prop is always completely ignored. The component constructs it from scratch before passing it down to the `<a>` element.

Reply:
I think it's fine. One less thing to think about as a consumer.